src/algebra/covariant_and_contravariant.lean

```diff
@@ -121,8 +121,8 @@ begin
     exact h a⁻¹ bc }
 end
 
-lemma covconv [group N] [cov : covariant_class N N (*) r] : contravariant_class N N (*) r :=
-{ covtc := λ a b c bc, group.covariant_iff_contravariant.mp cov.covc _ bc }
+lemma elimonv [group N] [cov : covariant_class N N (*) r] : contravariant_class N N (*) r :=
```

Comment:
wrong copy-paste

Reply:
Thanks for catching this!  And also for approving the PR!